.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{

```

Comment:
Please remove this file

Reply:
There must still be a configuration issue with GitHub because I'm not doing anything specific in this regard... I'm just using github.dev to report my modifications to the Dlib master branch. This file is apparently being added automatically...
In the meantime, I manually deleted it.